lib/rubocop/cop/layout/closing_parenthesis_indentation.rb

```diff
@@ -113,6 +121,26 @@ def check(node, elements)
                                         right_paren))
         end
 
+        def check_for_no_elements(node)
+          left_paren = node.loc.begin
+          right_paren = node.loc.end
+          return unless left_paren
```

Comment:
Is this guard condition necessary? The tests will pass even without this.

Reply:
Oh, the guard was unnecessary... I've fixed.